app/src/main/java/ca/bc/gov/bchealth/ui/healthrecord/labtest/LabTestScreen.kt

```diff
@@ -0,0 +1,394 @@
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material.Divider
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.ExperimentalComposeUiApi
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.input.pointer.pointerInput
+import androidx.compose.ui.platform.LocalSoftwareKeyboardController
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.SpanStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextDecoration
+import androidx.compose.ui.unit.dp
+import ca.bc.gov.bchealth.R
+import ca.bc.gov.bchealth.compose.BasePreview
+import ca.bc.gov.bchealth.compose.MyHealthTypography
+import ca.bc.gov.bchealth.compose.bold
+import ca.bc.gov.bchealth.compose.theme.HealthGatewayTheme
+import ca.bc.gov.bchealth.compose.theme.bannerBackgroundBlue
+import ca.bc.gov.bchealth.compose.theme.blue
+import ca.bc.gov.bchealth.compose.theme.darkText
+import ca.bc.gov.bchealth.compose.theme.dividerGrey
+import ca.bc.gov.bchealth.compose.theme.green
+import ca.bc.gov.bchealth.compose.theme.primaryBlue
+import ca.bc.gov.bchealth.compose.theme.red
+import ca.bc.gov.bchealth.ui.comment.CommentsSummary
+import ca.bc.gov.bchealth.ui.comment.CommentsSummaryUI
+import ca.bc.gov.bchealth.ui.component.HGSmallOutlinedButton
+import ca.bc.gov.bchealth.ui.custom.DecorativeImage
+import ca.bc.gov.bchealth.ui.custom.MyHealthClickableText
+import ca.bc.gov.bchealth.ui.healthrecord.labtest.LabTestDetailViewModel.Companion.ITEM_VIEW_PDF
+import ca.bc.gov.bchealth.ui.healthrecord.labtest.LabTestDetailViewModel.Companion.ITEM_VIEW_TYPE_LAB_ORDER
+import ca.bc.gov.bchealth.ui.healthrecord.labtest.LabTestDetailViewModel.Companion.ITEM_VIEW_TYPE_LAB_TEST
+import ca.bc.gov.bchealth.ui.healthrecord.labtest.LabTestDetailViewModel.Companion.ITEM_VIEW_TYPE_LAB_TEST_BANNER
+import ca.bc.gov.bchealth.utils.orPlaceholderIfNullOrBlank
+import ca.bc.gov.bchealth.widget.CommentInputUI
+import ca.bc.gov.common.BuildConfig
+
+@OptIn(ExperimentalComposeUiApi::class)
+@Composable
+fun LabTestScreen(
+    uiState: LabTestDetailUiState,
+    onClickViewPdf: () -> Unit,
+    onClickFaq: () -> Unit,
+    onClickComments: (CommentsSummary) -> Unit,
+    commentsSummary: CommentsSummary?,
+    onSubmitComment: (String) -> Unit,
+) {
+    val keyboardController = LocalSoftwareKeyboardController.current
+
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .pointerInput(Unit) {
+                detectTapGestures(onPress = { keyboardController?.hide() })
+            },
+    ) {
+        LabTestUi(uiState, onClickViewPdf, onClickFaq, onClickComments, commentsSummary)
```

Comment:
I would suggest to keep common naming, 
for example HomeScreen has a internal function called HomeScreenContent.
same way LabtTestScreen should have LabtTestScreenContent.

Reply:
done